src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -160,7 +160,7 @@ export const Map: React.FC<PropsType> = ({
 
   const { partygoersBySeat, isSeatTaken } = usePartygoersbySeat({
     venueId,
-    partygoers,
+    partygoers: recentVenueUsers,
   });
```

Comment:
[OOS] May be worth us renaming this hook something that doesn't use 'partygoers' in future.